Source/TCPError.swift

```diff
@@ -30,17 +30,17 @@ public enum TCPError: ErrorProtocol {
     case operationTimedOut(description: String, data: Data)
     case closedSocket(description: String)
 
-    static func lastReceiveErrorWithData(source: Data, bytesProcessed: Int) -> TCPError {
+    static func lastReceiveError(withData source: Data, bytesProcessed: Int) -> TCPError {
```

Comment:
`TCPError` will be most likely removed from the `TCP`, so it's better to discuss this in https://github.com/Zewo/OS/pull/1